packages/bento-design-system/src/Field/Field.css.ts

```diff
@@ -5,19 +5,25 @@ import { vars } from "../vars.css";
 
 export const readOnlyBackground = createVar();
 
+const notDisabled = ":not(:disabled):not([disabled])";
+
 export const inputRecipe = strictRecipe({
   base: [
     {
       "::placeholder": {
         color: vars.textColor.textSecondary,
       },
       selectors: {
+        "&:disabled, &[disabled]": {
+          background: vars.backgroundColor.backgroundPrimary,
+        },
         "&:disabled::placeholder": {
           color: vars.textColor.textDisabled,
         },
-        "input&:read-only, textarea&:read-only, &.readOnly, &[readonly]": {
-          background: readOnlyBackground,
-        },
+        [`input&:read-only${notDisabled}, textarea&:read-only${notDisabled}, &.readOnly${notDisabled}, &[readonly]${notDisabled}`]:
+          {
+            background: readOnlyBackground,
+          },
```

Comment:
@gabro  not sure if there's a better, less verbose way of doing this. The previous code was running correctly, but was generating build errors.